### ISSUE TITLE
Script to Import domain pricing

### ISSUE THREAD

**Issue by thomashoeky** [openprovider/OP-WHMCS7#41]
Hello,

It would be nice to have the ability to import pricing from openprovider.
I have made a script myself to do this but would like to see it in openprovider's own module and it will probably work even better :)
https://github.com/thomashoeky/opsync
You can use any code you want, if needed

**Comment by wmetge** [comment]
Hi @thomashoeky , thank you very much for sharing your code. We definitely are planning on including  a price sync tool in the near future, but until it's released, we'll keep this issue open so other users can benefit from your script.

**Comment by solucionesuno** [comment]
hello @thomashoeky 

thansk for share it!

where i have to put the script and how can i run it?

**Comment by thomashoeky** [comment]
@solucionesuno Im working on a new version since this version currently on my page is really unstable. Will @ you when it's done :)

**Comment by solucionesuno** [comment]
> @solucionesuno Im working on a new version since this version currently on my page is really unstable. Will @ you when it's done :)

thanks!!

**Comment by solucionesuno** [comment]
Hello, no news aboute this featured?

**Comment by Nova-ITT** [comment]
hi, how to install plis?

**Comment by soberoctober** [comment]
@Nova-ITT 

If you're referring to the script, then it cannot be installed per se, but rather has to be executed from an environment that has access to your WHMCS' database.

**Comment by solucionesuno** [comment]
Lets Go guys, Dondominio.com has a really good addon for this. 

if we can not import prices,... is not posisble sell all extesions...

**Comment by willdashwood** [comment]
I developed my own script and have been steadily improving it over the years. I'm still waiting on some changes to the OpenProvider API to perfect it but my contact at OP (Inna Avruneva) hasn't responded in a long time so I've kind of lost hope on that front. That said, I'm happy to share what I've created.

**Comment by soberoctober** [comment]
TLD pricing sync feature has been released in [version 3.3](https://github.com/openprovider/OP-WHMCS7/releases/tag/v3.3).
